src/main/groovy/com/marklogic/gradle/MarkLogicPlugin.groovy

```diff
@@ -181,6 +183,10 @@ class MarkLogicPlugin implements Plugin<Project> {
         project.task("mlWatch", type: WatchTask, group: modulesGroup, description: "Run a loop that checks for new/modified modules every second and loads any that it finds")
         project.task("mlDeleteModuleTimestampsFile", type: DeleteModuleTimestampsFileTask, group: modulesGroup, description: "Delete the properties file in the build directory that keeps track of when each module was last loaded")
 
+        String schemasGroup = "ml-gradle Schemas"
+        project.task("mlLoadSchemas", type: LoadSchemasTask, group: schemasGroup, description: "Loads special-purpose data into the schemas database (XSD schemas, Inference rules, and [MarkLogic 9] Extraction Templates)")
+        project.task("mlReloadSchemas", type: LoadSchemasTask, dependsOn: ["mlClearSchemasDatabase", "mlLoadSchemas"], group: schemasGroup, description: "Clears schemas database then loads special-purpose data into the schemas database (XSD schemas, Inference rules, and [MarkLogic 9] Extraction Templates)")
```

Comment:
For mlReloadSchemas, you actually don't want a type - look at mlReloadModules as an example. 

Also, the order of tasks in dependsOn doesn't force when Gradle will invoke the tasks. To ensure that mlLoadSchemas always runs after mlClearSchemasDatabase, you do this:

project.task("mlLoadSchemas", type....).mustRunAfter("mlClearSchemasDatabase")

I think mlLoadSchemas has to be defined after mlSchemasDatabase in order for mustRunAfter to work properly.